systests/cluster/test_remote_account.py

```diff
@@ -509,6 +511,29 @@ def test_monitor_log_exception(self):
             end = time.time()
             assert end - start > timeout, "Should have waited full timeout period while monitoring the log"
 
+    @cluster(num_nodes=1)
+    def test_kill_process(self):
+        """Tests that kill_process correctly works"""
+
+        def get_pids():
+            pid_cmd = "ps ax | grep -i nc | grep -v grep | awk '{print $1}'"
+
+            return list(node.account.ssh_capture(pid_cmd, callback=int))
+
+        node = self.account_service.nodes[0]
+
+        # Run TCP service using netcat
+        node.account.ssh_capture("nohup nc -l -p 5000 > /dev/null 2>&1 &")
```

Comment:
```suggestion
        node.account.ssh_capture("nohup nc -l -p 5000 > /dev/null 2>&1 &; disown")
```

I've encountered a behavior change on EL8 Platforms (CentOS-8 and RHEL8) where the SSHD daemon may terminate the shell, but the SSHD "session" for that connection remains open for the remainder of